buildconfig/stubs/pygame/rect.pyi

```diff
@@ -158,6 +171,12 @@ class Rect(Collection[int]):
     ) -> bool: ...
     def collidelist(self, rect_list: Sequence[RectValue]) -> int: ...
     def collidelistall(self, rect_list: Sequence[RectValue]) -> List[int]: ...
+    def collideobjectsall(
+        self, objects: Sequence[_T], key: Optional[Callable[[_T], RectValue]]
+    ) -> List[_T]: ...
+    def collideobjects(
+        self, objects: Sequence[_T], key: Optional[Callable[[_T], RectValue]]
```

Comment:
Ah this should take `key: Optional[Callable[[_T], RectValue]] = None` for the argument because `None` is the default value. Same with the other function

Reply:
@ankith26 sorry for the late reply, mail about this notification must have gone unread with all the other changes that where made in the last 3 weeks (200+ notifications, you all were busy). Updated as suggested.